airflow/dag_fetcher.py

```diff
@@ -0,0 +1,290 @@
+# -*- coding: utf-8 -*-
```

Comment:
please do not put this in the root dir but in something like `airflow/dag/fetcher/filesystem.py`